src/main/java/game/race/model/InputView.java

```diff
@@ -0,0 +1,55 @@
+package game.race.model;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String ASK_VEHICLE_COUNT = "자동차 대수는 몇 대 인가요?";
+    private static final String ASK_TRY_COUNT = "시도할 회수는 몇 회 인가요?";
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```
나머지도 대문자로 작성하셨는데 이부분도 일관성있게 대문자가 좋지 않나요?